From 7b7cc11d29072222b90a74712196ef55ba9298f3 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Fri, 20 Feb 2026 09:15:05 +0200
Subject: [PATCH] balena: iot-gate-imx8plus: Disable BINMAN

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 board/compulab/plat/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/board/compulab/plat/Kconfig b/board/compulab/plat/Kconfig
index 1b763e0e721..ef72efd74aa 100644
--- a/board/compulab/plat/Kconfig
+++ b/board/compulab/plat/Kconfig
@@ -43,7 +43,6 @@ config TARGET_SOM_IMX8M_PLUS
 config TARGET_IOT_GATE_IMX8PLUS
 	bool "CompuLab iot-gate-imx8plus"
 	select IMX8MP
-	select BINMAN
 	select SUPPORT_SPL
 	select IMX8M_LPDDR4
 
-- 
2.34.1

